governator-jersey/build.gradle

```diff
@@ -14,6 +14,10 @@ dependencies {
 
     compileOnly 'javax.servlet:javax.servlet-api:3.0.1'
 
+    implementation "javax.xml.bind:jaxb-api:2.3.1"
```

Comment:
I understand that jaxb was removed in Java 11, but where in governator code is it being used, or which of it's dependencies uses it?

Reply:
Unit tests.  If you run the tests they pass, but complain about the missing classes.  I'm guessing the test is broken, but appears passing.